cv-19-i-need-help-test/Gateways/V1/INeedHelpGatewayTest.cs

```diff
@@ -10,24 +14,114 @@ namespace CV19INeedHelpTest.Gateways.V1
     [TestFixture]
     public class INeedHelpGatewayTest
     {
-        [TestCase]
-        public void InsertMethodShouldNotBeCalledWithoutParameter()
+        private INeedHelpGateway _classUnderTest;
+        private readonly Fixture _fixture = new Fixture();
+        private Cv19SupportDbContext _context;
+        private string _connectionString;
+        
+        [SetUp]
+        public void SetUp()
         {
-//            string connectionString = string.Empty;
-//            var data = new OrganisationsNeedingVolunteers() ;
-//            IOrganisationVolunteerGateway classUnderTest = new OrganisationVolunteerGateway(connectionString);
-//            var resp = classUnderTest.Insert(data);
-//            Assert.AreEqual(1,resp);
+            _connectionString = Environment.GetEnvironmentVariable("CV_19_DB_CONNECTION");
```

Comment:
If you inherit from the DatabaseTests class, it will handle a lot of the setup for you. You will just need to add in the extra teardown to clear the other tables.